bzt/engine/engine.py

```diff
@@ -705,6 +706,8 @@ def eval_env(self):
         """
         envs = self.config.get(SETTINGS, force_set=True).get("env", force_set=True)
         envs[TAURUS_ARTIFACTS_DIR] = self.artifacts_dir
+        envs[TAURUS_USER_CONFIGS] = ','.join(map(str, self.config.get('user-configs', force_set=True)))
+        self.log.warning("Env vars: " + envs[TAURUS_USER_CONFIGS])
```

Comment:
we prefer to use _info_ level for such kind of message

Reply:
Removed this logging, I added this just for my local debug needs and forgot to remove before commit